templates/html/index.html

```diff
@@ -4,9 +4,18 @@
   <meta charset="UTF-8">
   <title>{{asyncapi.info().title()}} {{asyncapi.info().version()}} documentation</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link href="css/tailwind.min.css" rel="stylesheet">
-  <link rel="stylesheet" href="css/atom-one-dark.min.css" />
-  <link rel="stylesheet" href="css/main.css" />
+
```

Comment:
I've seen you're changing this file permissions to 755. Is there a reason for this? Wasn't it on purpose?